Create rule S6561: Do not use DateTime.Now for benchmarking or timing operations (#1711)
SonarSource/rspec: rules/S6561/introduction.adoc

> The rule targets the use of multiple `DateTime.Now` calls, followed by some arithmetic operations, to measure the elapsed time between actions.

Reviewer: ```suggestion
The rule targets the use of multiple `DateTime.Now` calls in the same code block, used to measure the elapsed time between actions.
```

Reviewer: code block = method I would say.

> The rule targets the use of multiple `DateTime.Now` calls, followed by some arithmetic operations, to measure the elapsed time between actions.

Reviewer 2: The rule targets the use of multiple DateTime.Now calls, that are part of arithmetic operations.
